src/runtime/Converter.cs

```diff
@@ -980,5 +981,11 @@ public static PyObject ToPython(this object? o)
             if (o is null) return Runtime.None;
             return Converter.ToPython(o, o.GetType()).MoveToPyObject();
         }
+
+        public static PyObject ToPythonAs<T>(this T? o)
```

Comment:
`ToPython<T>` would work as well, right?

Reply:
Not really, that would be code incompatibility, because 

```csharp
IWhoops value = new Foo();
var py = value.ToPython();
```

if we name the function `ToPython` it would be called here instead of the original one without any code modification, and the result will be different (e.g. interface representation instead of `Foo` object).